config/i18n-tasks.yml

```diff
@@ -1,3 +1,8 @@
+base_locale: en
+data:
```

Comment:
@teoljungberg I took pieces from the [default yml](https://github.com/glebm/i18n-tasks/blob/master/templates/config/i18n-tasks.yml) in the i18n-tasks gem